tests/integration/authentication.test.ts

```diff
@@ -94,8 +94,8 @@ describe('Authentication', () => {
 
     beforeEach(async () => navigateThroughToSecretKeyPage(), 15_000);
 
-    test('it does not let you proceed when entering an incorrect seed phrase', async done => {
-      const nonsenseRhymingSeed = 'You might encounter some delays, if you forget your seed phrase';
+    test('it does not let you proceed when entering an incorrect Secret Key', async done => {
+      const nonsenseRhymingSeed = 'It does not happen frequently, but dont forget your Secret Key';
```

Comment:
Very important that we keep this rhyming pattern going as things continue to evolve.

Reply:
As you can see, it was a low-effort rhyme 😂